package.json

```
@@ -55,7 +55,7 @@
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^7.0.0 || ^8.0.0",
-    "@openedx/paragon": "^22.0.0",
+    "@openedx/paragon": "^23.0.0",
```

Could this be done as a non-breaking change?

If

```suggestion
    "@openedx/paragon": "^22.0.0 || ^23.0.0",
```

works that would be awesome.